felix/bpf/proxy/syncer.go

```diff
@@ -780,7 +780,12 @@ func (s *Syncer) updateService(skey svcKey, sinfo k8sp.ServicePort, id uint32, e
 		cnt++
 	}
 
-	if err := s.writeSvc(sinfo, id, cnt, local, 0); err != nil {
+	flags := uint32(0)
+	if sinfo.InternalPolicyLocal() {
```

Comment:
Should we do this for clusterIP?

Reply:
This is the path taken by the cluster IP and yes, we need to https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/